packages/astro/src/core/dev/template/error.ts

```diff
@@ -1,14 +1,23 @@
 import { encode } from 'html-entities';
 
 interface ErrorTemplateOptions {
+  /** a short description of the error */
+  message: string;
+  /** information about where the error occurred */
+  stack?: string;
+  /** HTTP error code */
   statusCode?: number;
+  /** HTML <title> */
   tabTitle: string;
+  /** page title */
   title: string;
-  message: string;
+  /** show user a URL for more info or action to take */
+  url?: string;
 }
 
 /** Display internal 404 page (if user didn’t provide one) */
-export function errorTemplate({ title, message, statusCode, tabTitle }: ErrorTemplateOptions): string {
+export function errorTemplate({ title, url, message, stack, statusCode, tabTitle }: ErrorTemplateOptions): string {
+  let error = url ? message.replace(url, '') : message;
```

Comment:
This is a bit of a hack—we pass a `url` property on the error to format a link. Open to suggestions on making it better